src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -405,13 +409,13 @@ public void run() {
   private class ProduceServiceThreadFactory implements ThreadFactory {
 
     private final AtomicInteger _threadId = new AtomicInteger();
-    public Thread newThread(Runnable r) {
+    public Thread newThread(@SuppressWarnings("NullableProblems") Runnable r) {
       return new Thread(r, _name + "-produce-service-" + _threadId.getAndIncrement());
     }
   }
 
   private class HandleNewPartitionsThreadFactory implements ThreadFactory {
-    public Thread newThread(Runnable r) {
+    public Thread newThread(@SuppressWarnings("NullableProblems") Runnable r) {
```

Comment:
This seems outside of rebranding.  Would suggest that it gets split out into another commit for history hygiene.

Reply:
sgtm